src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
```

Comment:
Would give you a more thorough code review, but 
https://github.com/briancodex/react-todo-app-v1/tree/master
https://www.youtube.com/watch?v=E1E08i2UJGI